### DISCUSSION TITLE
Can I have the page heading different compared to what it says in the navbar?

### DISCUSSION THREAD

**Discussion by Ravencentric** [retypeapp/retype#518] [Q&A] [answered]
Something like this:

![image](https://user-images.githubusercontent.com/78981416/227599972-50a32021-9c10-4f36-b546-17cdbf61c0c6.png)

**Comment by DannyBen** [comment, marked as answer]
Have you tried the [`label`](https://retype.com/configuration/page/#label) front matter directive?

```markdown
---
label: Navbar Title
---

# Page Title

Rest of the page...
```

**Reply by DannyBen** [reply]
@geoffreymcgill - the documentation for [`title`](https://retype.com/configuration/page/#title) hints that this option is discouraged, and provided for legacy support. Since it is useful to allow creating a different navbar title, shouldn't the documentation be changed to embrace this feature instead of discouraging it?

**Reply by DannyBen** [reply]
Sorry - I remembered there is `label` instead of `title` - perhaps the documentation tip should mention `label` as an alternative.

**Reply by Ravencentric** [reply]
That didn't help, you can see here https://github.com/Ravencentric/retype-test/blob/main/README.yml

**Reply by geoffreymcgill** [reply]
Your recommendation to use `label` is the best approach.

btw, in Retype v3.0, you will also be able to set `meta.title` which allows for writing a custom `<title>Your custom title here</title>` tag. Not to be confused with `label` and `title`. I'd still recommend not using the `title` config as it is better to focus on writing a proper descriptive primary H1 `# Page title` for your content. 

Hope this helps.

**Reply by geoffreymcgill** [reply]
Hi @Ravencentric, 

If you are separating the frontmatter page metadata into its own `.yml` file, then you need to remove the `---` separators. 

If you revise your `README.yml` to the following, it will work:

```yml
label: Awesome *Arrs NEW
order: -1
author: Ravencentric
```

Also, it's best to remove the `title` config, although it will still work just fine with it in there. Just better to keep things clean.

Another option is to delete your `README.yml` file and add the metadata to the top of the `README.md`. For example:

```md
---
label: Awesome *Arrs NEW
order: -1
author: Ravencentric
---
# Awesome *Arr

[-![Awesome](https://awesome.re/badge.svg)](https://awesome.re)
```

Hope this helps.

**Reply by Ravencentric** [reply]
Yes, that works. Thank you both for the help!
